config/pre-push-recommended.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
 ./gradlew -q \
+	ktlint --args="-F" \
```

Comment:
This may confuse people when their code check-in succeeds but they have new changes. I guess it may be okay if it's safe enough to automate.

Reply:
Definitely did not mean to push this--whoops! Was just testing something for my setup 😅